inventory/byo/hosts.localhost

```diff
@@ -0,0 +1,21 @@
+#bare minimum hostfile
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ansible_ssh_user=root
+debug_level=2
+openshift_deployment_type=origin
+openshift_release=3.7
```

Comment:
Seeing as 3.7 packages aren't widely available yet, we should use 3.6.

Reply:
lets see if I can just make it readily available